docs/plugins/discovery-ec2.asciidoc

```diff
@@ -195,6 +195,20 @@ as valid network host settings:
 |`_ec2_` |equivalent to _ec2:privateIpv4_.
 |==================================================================
 
+[[discovery-ec2-caching]]
+===== EC2 Node Caching
+
+Once the discovery mechanism found a list of hosts, the result is cached for a certain
```

Comment:
I would not add a specific section for this but I'd add it to this table:

```
The following are a list of settings (prefixed with `discovery.ec2`) that can further control the discovery:

`groups`::

    Either a comma separated list or array based list of (security) groups.
    Only instances with the provided security groups will be used in the
    cluster discovery. (NOTE: You could provide either group NAME or group
    ID.)

`host_type`::

    The type of host type to use to communicate with other instances. Can be
    one of `private_ip`, `public_ip`, `private_dns`, `public_dns`. Defaults to
    `private_ip`.

`availability_zones`::

    Either a comma separated list or array based list of availability zones.
    Only instances within the provided availability zones will be used in the
    cluster discovery.

`any_group`::

    If set to `false`, will require all security groups to be present for the
    instance to be used for the discovery. Defaults to `true`.

`ping_timeout`::

    How long to wait for existing EC2 nodes to reply during discovery.
    Defaults to `3s`. If no unit like `ms`, `s` or `m` is specified,
    milliseconds are used.

`node_cache_time `::

    How long the list of hosts is cached to prevent further requests to the AWS API. 
    Defaults to `10s`.
```